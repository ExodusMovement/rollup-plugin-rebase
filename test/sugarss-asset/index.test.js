import { bundle, clean, list, read } from "../util"

const root = __dirname

test("Sugar SS Asset", async () => {
  await bundle(root, "index.js", "output/index.js")

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/dtlVeIbo.pcss")).toMatchSnapshot()

  await clean(root, "output")
})
