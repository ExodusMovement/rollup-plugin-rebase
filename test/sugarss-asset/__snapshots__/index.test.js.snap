// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sugar SS Asset 1`] = `
"bXipeBrs.png
bsdudxaF.md
dtlVeIbo.pcss
dtlVeIbo.pcss.map
index.js"
`;

exports[`Sugar SS Asset 2`] = `
"import content from './bsdudxaF.md';
import css from './dtlVeIbo.pcss';

function index() {
  return content + css
}

export default index;
"
`;

exports[`Sugar SS Asset 3`] = `
"body {
  font-size: 20px;
  background: url(./bXipeBrs.png) top left repeat
}

/*# sourceMappingURL=dtlVeIbo.pcss.map */"
`;
